sdk/playwrighttesting/microsoft-playwright-testing/src/utils/reporterUtils.ts

```diff
@@ -158,7 +158,7 @@ class ReporterUtils {
     testResult.webTestConfig = {
       jobName: jobName,
       projectName: test.parent.project()!.name,
```

Comment:
these are also compulsory fields; we should check the behavior when these are not set in the playwright config. Looks like, if project is not present in the config, this can throw NPE.

Reply:
checked, it doesnot throw any error as it is not undefined , we get it as an empty string if project is not set in config.